config/locales/ar.yml

```diff
@@ -33,11 +33,19 @@ ar:
     status_count_after:
       one: منشور
       other: منشورات
+      zero: منشورات
+      two: منشورات
+      few: منشورات
+      many: منشورات
     status_count_before: نشروا
     terms: شروط الخدمة
     user_count_after:
       one: مستخدم
       other: مستخدمين
+      zero: مستخدمين
+      two: مستخدمين
+      few: مستخدمين
+      many: مستخدمين
```

Comment:
It seems that the Weblate has only items "one" and "other" in Arabic.
https://weblate.joinmastodon.org/translate/mastodon/backend/ar/?checksum=c02537acd5ed53a4
I don't know how to solve it…

Reply:
Noticed that for the other languages which have different plurals too, I'm going to check the issues on weblate to see if it's configuration or what can be done